lighthouse-core/test/audits/long-tasks-test.js

```diff
@@ -50,18 +50,21 @@ function generateTraceWithLongTasks({count, duration = 200, withChildTasks = fal
   }
   return createTestTrace({
     topLevelTasks: traceTasks,
+    navigationStart: BASE_TS,
   });
 }
 
 describe('Long tasks audit', () => {
+  const devtoolsLog = networkRecordsToDevtoolsLog([{url: TASK_URL}]);
+
   it('should pass and be non-applicable if there are no long tasks', async () => {
     const artifacts = {
       traces: {defaultPass: generateTraceWithLongTasks({count: 0})},
       devtoolsLogs: {defaultPass: devtoolsLog},
     };
     const result = await LongTasks.audit(artifacts, {computedCache: new Map()});
-    expect(result.score).toBe(1);
     expect(result.details.items).toHaveLength(0);
+    expect(result.score).toBe(1);
```

Comment:
FYI @Beytoven I moved these down so that when there's a failure you get the debug information about what's inside `details.items` instead of `expected: 1, received: 0` :)